### PR TITLE
Initial configuration of repo

### PR DESCRIPTION
**Pull request by drewoldag**
Applying the LINCC frameworks Python project template
Added `SinusoidalLightcurve` source, test, simple notebook and documentation page
Provided scaffolding for additional documentation as well as placeholder documentation to be filled later.

### CODE REVIEW

**Comment by drewoldag** [comment]
Note that the CI will initially fail because the version of `sorcha` that is available on PyPI needs to be updated. Once this PR (https://github.com/dirac-institute/sorcha/pull/493) is merged, we should be in a good place to increment the version number and then the CI here will pass. 

Testing was performed locally using `sorcha` installed from github with pip. (Not what we would want in production)

File: src/sorcha_community_utils/sinusoidal/sinusoidal_lightcurve.py
Comment (mschwamb) [inline src/sorcha_community_utils/sinusoidal/sinusoidal_lightcurve.py:1]: Maybe we're not going to have a lot of classes, but I think it's worth have an lc and activity directory and then you have your different developed classes in there so if I go to activity these are prescriptions for cometary activity and if I go into the lc directory it's prescriptions of rotational light curve effects. 

So I think this should be in lc/sinuisoidal/sinuisoidal_lightcurve.py 

I am definitely open to other suggestions.

File: tests/sinusoidal/test_sinusoidal.py
Comment (mschwamb) [inline tests/sinusoidal/test_sinusoidal.py:1]: Same thing as above. Should the tests also be split by light curve and activity?

**Review by mschwamb — CHANGES_REQUESTED** [review]

File: src/sorcha_community_utils/sinusoidal/sinusoidal_lightcurve.py
Comment (drewoldag) [reply src/sorcha_community_utils/sinusoidal/sinusoidal_lightcurve.py:1]: I'm up for extra organizing, so this is no problem to implement.

**Review by mschwamb — APPROVED** [review]

**Pull request merged**